examples/sorting/experiments.yml

```diff
@@ -15,7 +15,5 @@ instances:
 experiments:
   - name: insertion-sort
     args: ['./sort.py', '--algo=insertion-sort', '@INSTANCE@']
-    output: stdout
```

Comment:
We should not change the example here (its output is needed to make an evaluation script work).

Reply:
I reverted the changes to the `experiment.yml`.